web/app/scripts/src/services/proportion-field.es.js

```diff
@@ -55,6 +52,16 @@ module.service(moduleProperties.componentName, [
             });
         };
 
+        this.getChangedFieldIndex = function (newModel, oldModel) {
+            const newValues = Object.values(newModel);
+            const oldValues = Object.values(oldModel);
+
+            return newValues.reduce((sum, element, index) => {
```

Comment:
Using `reduce` is a neat solution to the problem when `findIndex` is not available.